skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -104,18 +104,52 @@ def test_lr_callback_batch_steps_correctly(
             policy,
             kwargs,
     ):
-        num_examples = 1000
         batch_size = 100
-        max_epochs = 2
+        max_epochs = 1
```

Comment:
Wouldn't `max_epochs == 2` be a better test for this since it also covers the case of multiple epochs?

Reply:
The original intention was to use `fit` and `partial_fit` to simulate two epochs.

This PR has been updated to used `max_epochs=2` and use one fit`. The `test_lr_callback_batch_steps_correctly_fallback` tests for partial fit and that the fallback works.